content/en/post/zheyuan-wei-iot/index.md

```diff
@@ -0,0 +1,34 @@
+---
+# Documentation: https://wowchemy.com/docs/managing-content/
+
+title: "Wei Zheyuan presented his research at IPSJ SIG Internet and Operation Technology (IOT)"
+subtitle: ""
+summary: ""
+authors: ["Wei Zheyuan"]
+tags: ["Multi-Path QUIC","SRv6","Routing"]
+categories: []
+date: 2024-09-21T14:33:58+09:00
+lastmod: 2024-09-21T14:33:58+09:00
+featured: false
+draft: false
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
+  caption: ""
+  focal_point: ""
+  preview_only: false
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+projects: []
+---
+
+Wei Zheyuan, a second-year master's student in our laboratory, presented his research at [IPSJ SIG Internet and Operation Technology (IOT)](https://www.iot.ipsj.or.jp/).
+
+
+He introduced his research titled "A Proposal of Path Selection Method by SRv6 for Optimizing Multipath QUIC Performance",which create a model that optimizing Multipath QUIC transmission performance by SRv6, and received a Student Research Award.
```

Comment:
Should be:
He introduced his research titled "A Proposal of Path Selection Method by SRv6 for Optimizing Multipath QUIC Performance," which proposed a model to optimize Multipath QUIC transmission performance by SRv6, and received a Student Research Award.